cost-model/src/cost_model.rs

```diff
@@ -155,70 +155,103 @@ impl CostModel {
         transaction: &impl TransactionWithMeta,
         feature_set: &FeatureSet,
     ) -> (u64, u64, u64) {
-        let mut programs_execution_costs = 0u64;
```

Comment:
I would prefer if this function stays basically as-is but renamed to something like `get_transaction_cost_without_minimal_builtin_cus` to keep the diff smaller. Then a wrapper function `get_transaction_cost` can call this function if the new feature gate isn't active yet.

Reply:
changed